# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unicap/glib.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- unicap-0.2.22/libucil/ucil_alsa.h.vanilla	2008-05-12 17:12:31.000000000 +0200
+++ unicap-0.2.22/libucil/ucil_alsa.h	2008-05-12 17:12:34.000000000 +0200
@@ -29,7 +29,7 @@
 #ifndef   	UCIL_ALSA_H_
 # define   	UCIL_ALSA_H_
 
-#include <glib-2.0/glib.h>
+#include <glib.h>
 
 void  *ucil_alsa_init( char *dev, unsigned int rate );
 long   ucil_alsa_fill_audio_buffer( void *_data );
--- unicap-0.2.22/libucil/ucil_alsa.c.vanilla	2008-05-12 17:12:41.000000000 +0200
+++ unicap-0.2.22/libucil/ucil_alsa.c	2008-05-12 17:12:45.000000000 +0200
@@ -30,7 +30,7 @@
 
 #include <stdlib.h>
 #include <asoundlib.h>
-#include <glib-2.0/glib.h>
+#include <glib.h>
 /* #if UCIL_DEBUG */
 #define DEBUG
 /* #endif */
